### ISSUE TITLE
Add configuration policy to new project flows

### ISSUE THREAD

**Issue by olensmar** [kubeshop/monokle#4091]
When creating a new project I want to specify which validation plugins that should be enabled - instead of having to do that later. Suggest we add a checklist to the corresponding forms/panels - one item for each validation plugin (individual rules can be configured later) - default to enable:
- links
- schema (select k8s version?)
- common practices

**Comment by julioramosest** [comment]
@olensmar agree with adding a multiselect list to corresponding forms for each one of the starting options. Together with some easy info/trust text label or similar, so beginner users don't get lost/frustrated. I'll work on something

**Comment by julioramosest** [comment]
Find ux proposal here: https://www.figma.com/file/V9FLU3sgj5oIwDhAIRNyIb/Policy?type=design&node-id=840%3A5115&mode=design&t=U7DNymsDYCIFwelL-1

Per @WitoDelnat suggestions:
- We can leave the project creation as it is and just after creation user being launched to the policy wizard (also moving the policy tab to 2nd place instead of 3rd in the project view)
- This would provide a less cluttered, more step-like project creation, also bc gh bot activation in the same step would add extra complexity to project creation (related to issue https://github.com/kubeshop/monokle-saas/issues/1643 ).

**Comment by olensmar** [comment]
@julioramosest this was actually intended for Desktop - but the designs seem to for Cloud?

**Comment by julioramosest** [comment]
@olensmar yes I noticed that the other day, my bad. Let me rework it for Desktop

**Comment by julioramosest** [comment]
@olensmar @devcatalin find design and UX notes here: https://www.figma.com/file/V9FLU3sgj5oIwDhAIRNyIb/Policy?type=design&node-id=840%3A5115&mode=design&t=S7eKTkRRnRgiy28n-1 I believe this works for most new project cases, let's review individually if needed.

<img width="1440" alt="adding a policy setup to project creation 1" src="https://github.com/kubeshop/monokle/assets/97731654/8a3cc934-3f11-4b8d-8c5b-00bfa283c8cd">

<img width="1440" alt="adding a policy setup to project creation 2" src="https://github.com/kubeshop/monokle/assets/97731654/42f24700-c518-4ea3-8c93-47119d3154d2">

<img width="1440" alt="adding a policy setup to project creation 3" src="https://github.com/kubeshop/monokle/assets/97731654/7d137d7f-2ac6-4b53-8262-82579e7d43c7">

**Comment by julioramosest** [comment]
To Catalin request, see UI and UX notes for Cloud version of this feature (adding policy set up to project creation): https://www.figma.com/file/V9FLU3sgj5oIwDhAIRNyIb/Policy?type=design&node-id=934%3A3603&mode=design&t=p4yTV0dQU5m29tsO-1
